packages/browser-lite/src/schema_validator.ts

```diff
@@ -0,0 +1,166 @@
+import * as ajv from 'ajv';
+
+import { JsonSchema } from './types';
+
+export interface SchemaValidationResult {
+    success: boolean;
+    errors: string[];
+    fatal?: string;
+}
+
+export interface SchemaValidator {
+    orderValidator: (input: string) => SchemaValidationResult;
+    messageValidator: (input: string) => SchemaValidationResult;
+}
+
+/**
+ * Constructs a `SchemaValidator` object that is compatible with the `orderfilter` package.
+ * @param chainId The chainId of the current ethereum network
+ * @param exchangeAddress The exchange address that Mesh should recognize on the current network
+ * @param customOrderFilter An extension of the order schema that adds specific requirements to
+ *        orders.
+ */
+export function getSchemaValidator(
+    chainId: number,
+    exchangeAddress: string,
+    customOrderFilter?: JsonSchema,
+): SchemaValidator {
+    const chainIdSchema = {
+        $async: false,
+        $id: '/chainId',
+        const: chainId,
+    };
+
+    // NOTE(jalextowle): The current implementation of `zeroex.SignedOrder.MarhsalJSON`
```

Comment:
I don't think we should make this change here. It is potentially breaking depending on how people are interacting with Mesh.

I would not expect an `enum` with two values to result in a big performance hit anyways. Is that not the case?